deliverables/ruby_monk_problems/exec_time.rb

```diff
@@ -1,3 +1,6 @@
 def exec_time(proc)
-  # your code here
-end
+  time1 = Time.now
+  lambda {|proc| return proc}
```

Comment:
This isn't correct. Line #3 lambda isn't assigned to anything here and this doesn't accomplish anything. The time returned is wrong. Good effort though :+1: